text/0033-view-hierarch.md

```diff
@@ -0,0 +1,41 @@
+* Start Date: 2022-11-02
+* RFC Type: feature
+* RFC PR: [#33](https://github.com/getsentry/rfcs/pull/33)
+
+# Summary
+
+Allow SDKs to send View hierarch of the application UI.
+
+# Motivation
+
+Being able to see what the user was looking at is a good way to found out problems in the application. 
+This RFC proposes a way to send information from the UI state of an application, known as "view hierarch".
```

Comment:
I'd elaborate a bit more here, it's pretty straightforward but assumes everyone knows what a view hierarchy is and how this is a good thing, linking to the other PoCs that have screenshots, etc.